deploy/app.py

```diff
@@ -69,6 +69,11 @@
 resource_prefix = app.node.try_get_context('resource_prefix') or 'dataall'
 
 source = app.node.try_get_context('repository_source') or 'codecommit'
+repo_string = app.node.try_get_context('repo_string') or 'awslabs/aws-dataall'
+repo_connection_arn = app.node.try_get_context('repo_connection_arn')
```

Comment:
For future reference:  If the requested value is not defined,  app.node.try_get_context returns `None` in Python rather that raising an exception. So, the current implementation is correct.